content/blocksPages/terms-of-service.json

```diff
@@ -13,8 +13,8 @@
 THAT THE ARBITRATOR MAY AWARD RELIEF ONLY TO AN INDIVIDUAL CLAIMANT AND ONLY TO THE EXTENT NECESSARY TO PROVIDE RELIEF ON YOUR INDIVIDUAL CLAIM(S). ANY RELIEF AWARDED MAY NOT AFFECT OTHER TINA.IO USERS. YOU AND TINA.IO FURTHER AGREE THAT, BY ENTERING INTO THIS AGREEMENT, YOU AND TINA.IO ARE EACH WAIVING THE RIGHT TO A TRIAL BY JURY OR TO BRING, JOIN, OR PARTICIPATE IN A CLASS ACTION, COLLECTIVE ACTION, PRIVATE ATTORNEY GENERAL ACTION, OR OTHER REPRESENTATIVE PROCEEDING OF ANY KIND AS A PLAINTIFF OR CLASS MEMBER.\n\n**12.1 Assignment** This Agreement, and any rights and licenses granted hereunder, may not be transferred or assigned by you, but may be assigned by Tina.io without restriction. Any attempted transfer or assignment in violation hereof shall be null and void.\n\n**12.2 Notification Procedures and Changes to the Agreement** Tina.io may provide notifications, whether such notifications are required by law or are for marketing or other business related purposes, to you via email notice, written or hard copy notice, or through posting of such notice on our website, as determined by Tina.io in our sole discretion. Tina.io reserves the right to determine the form and means of providing notifications to our Users, provided that you may opt out of certain means of notification as described in this Agreement. Tina.io is not responsible for any automatic filtering you or your network provider may apply to email notifications we send to the email address you provide us. Tina.io may, in its sole discretion, modify or update this Agreement from time to time, and so you should review this page periodically. When we change the Agreement in a material manner, we will update the ‘last modified’ date at the top of this page and notify you that material changes have been made to the Agreement. Your continued use of the Service after any such change constitutes your acceptance of the new Terms of Service. If you do not agree to any of these terms or any future Terms of Service, do not use or access (or continue to access) the Service.\n\n**12.3 Entire Agreement/Severability** This Agreement, together with any amendments and any additional agreements you may enter into with Tina.io in connection with the Service, shall constitute the entire agreement between you and Tina.io concerning the Service. Except as otherwise stated in Section 12.2, if any provision of this Agreement is deemed invalid by a court of competent jurisdiction, the invalidity of such provision shall not affect the validity of the remaining provisions of this Agreement, which shall remain in full force and effect.\n\n**12.4 No Waiver** No waiver of any term of this Agreement shall be deemed a further or continuing waiver of such term or any other term, and Tina.io’s failure to assert any right or provision under this Agreement shall not constitute a waiver of such right or provision.\n\n**12.5 Contact** Please contact us at admin@ssw.com.au with any questions regarding this Agreement.\n",
```

Comment:
"Last Updated" needs to be updated?

Reply:
![image](https://github.com/tinacms/tina.io/assets/25432120/12e7e231-57d5-48ff-9b41-fbff96cb17a5)
Adding image for context

Reply:
Thanks guys - fixed